timely_beliefs/visualization/graphs.py

```diff
@@ -259,7 +259,7 @@ def source_vs_hour_chart(
                 "event_start:O",
                 timeUnit="hours",
                 axis=alt.Axis(domain=False, ticks=False, labelAngle=0),
-                scale=alt.Scale(domain=list(range(24))),
+                # scale=alt.Scale(domain=list(range(24))),
```

Comment:
Why is this commented out? Do we want to keep it?

Reply:
I had hoped to fix it, but was unable to. I believe it is a vega-lite regression. Just for future reference, I found [an .. interesting .. workaround](https://github.com/altair-viz/altair/issues/1892) for `timeUnit="hoursminutes"`, which works, but it doesn't work for `timeUnit="hours"`.